### ISSUE TITLE
Error for being unable to connect to k8s is convoluted

### ISSUE THREAD

**Issue by ewolinetz** [fabric8io/fluent-plugin-kubernetes_metadata_filter#16]
If the plugin is unable to connect to K8S, it doesn't print an error that is easy for users to interpret.

```
2016-03-03  03:32:39 +0000 [error]: unexpected error  error_class=Errno::ECONNREFUSED error=#<Errno::ECONNREFUSED:  Connection refused - connect(2)>
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:878:in `initialize'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:878:in `open'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:878:in `block in connect'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/timeout.rb:52:in `timeout'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:877:in `connect'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:1445:in `begin_transport'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:1402:in `transport_request'
  2016-03-03 03:32:39 +0000 [error]: /usr/share/ruby/net/http.rb:1376:in `request'
  2016-03-03 03:32:39 +0000 [error]: /opt/app-root/src/gems/kubeclient-0.4.0/lib/kubeclient/watch_stream.rb:20:in `each'
   2016-03-03 03:32:39 +0000 [error]:  /opt/app-root/src/gems/fluent-plugin-kubernetes_metadata_filter-0.16.0/lib/fluent/plugin/filter_kubernetes_metadata.rb:230:in  `start_watch'
   2016-03-03 03:32:39 +0000 [error]:  /opt/app-root/src/gems/fluent-plugin-kubernetes_metadata_filter-0.16.0/lib/fluent/plugin/filter_kubernetes_metadata.rb:140:in  `block in configure'
2016-03-03 03:32:47 +0000 [error]: fluentd main process died unexpectedly. restarting.
```

**Comment by jimmidyson** [comment]
Yeah, not nice. Happy for a PR to improve that.

**Comment by jimmidyson** [comment]
Closing - fixed in #17
